sdk/core/System.Net.ClientModel/api/System.Net.ClientModel.net6.0.cs

```diff
@@ -50,13 +50,15 @@ public partial class NullableResult<T> : System.Net.ClientModel.Result
         public virtual T? Value { get { throw null; } }
         public override System.Net.ClientModel.Core.MessageResponse GetRawResponse() { throw null; }
     }
-    public partial class RequestOptions : System.Net.ClientModel.Core.PipelineOptions
+    public partial class RequestOptions
```

Comment:
Add a method AddPolicy to make it easy to modify after copy constructor from PipelineOptions is called without requiring us to new and copy the PipelineOptions ReadOnlyMemory<Policy> to add to it.